src/vstart.sh

```diff
@@ -472,7 +472,7 @@ run() {
         if [ "$nodaemon" -eq 0 ]; then
             prun "$@"
         else
-            prunb ./ceph-run "$@" -f
+            prunb ${SRC_PATH}/ceph-run "$@" -f
```

Comment:
i'd suggest use `$CEPH_ROOT/src/ceph-run` instead of using the fact that `vstart.sh` is co-located with this script. more consistent this way. please see how we use `$CEPH_ROOT` in this script for more examples.